src/west/app/project.py

```diff
@@ -401,9 +401,24 @@ def do_add_parser(self, parser_adder):
             - cloned: "cloned" if the project has been cloned, "not-cloned"
               otherwise
             - clone_depth: project clone depth if specified, "None" otherwise
+            - groups: project groups, as a comma-separated list
+
+            INCLUDED PROJECTS
+            -----------------
```

Comment:
I have the suspicion that more users will execute `west help list` to find out why `<someproject>` is not in the list of printed project, compared to users wanting to know print formatting.

So consider if this section should move above `FORMAT STRINGS` section.

Reply:
> So consider if this section should move above `FORMAT STRINGS` section.

I've done this as part of a more general rework related to the other options.